angular/src/app/shared/angular-example/angular-example.service.ts

```diff
@@ -41,14 +41,14 @@ export class AngularExampleService {
 
   deleteCookies() {
     return this.start()
-      .setUri('/api/fusionauth/cookies')
+      .setUri('/api/fusionauth/logout')
```

Comment:
These "deleteCookies" and "setCookies" functions are not intended to be the Login and Logout APIs. 

Instead of modifying these, can you add new methods in this service for `login` and `logout`?

Reply:
Agreed.  I haven't thought about these in a while, but I think the `(set|delete)Cookies` methods where for some SPA specific things around getting tokens out of the JS and into `httponly` cookies.  They are related to, but not the same as login/out.  I think you can refer to one of the other fusionauth-clients (e.g. `fusionauth-javascript-client`) for `login` and `logout` methods.

Reply:
Great, that is what I remembered as well. @engineertdog  Can you update? Thanks.